python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -670,8 +665,7 @@ def getHourAngle(self, mjd, ra):
     @property
     def observatory(self):
         if self._observatory is None:
-            self._observatory \
-                = LsstSimMapper().MakeRawVisitInfoClass().observatory
+            self._observatory = LsstObservatory()
```

Comment:
I couldn't figure out if LSSTObservatory was a singleton or not (as it is being used in more than one file).  Is that handled through LsstSimMapper via LsstSimMapper().MakeRawVisitInfoClass().observatory?

Also, it looks like this is a place we still need obs_lsstSim is that right?

Reply:
> it looks like this is a place we still need obs_lsstSim is that right?
yes.